content/release-10-15-0/whatsnew-10-15-0.md

```diff
@@ -11,3 +11,9 @@ Release 10.15.0 includes the following new features or major feature enhancement
 In an effort to provide you a more consistent experience across the platform the **All devices** page has been re-shaped. It is now implemented in the data grid fashion you already are familiar with from **Subassets** and **Search** pages. This offers improved appearence in various viewport sizes, easier filtering and sorting, pagination option (only for users with global inventory access) and sets solid ground for further improvements.
 
 ![All devices](/images/release-notes/devmgmt-devices-alldevices.png)
+
+### Multiple LNS connection for Sigfox and Actility
+
+The Sigfox and Actility agents now support the configuration of multiple connections per tenant. One connection can be associated with multiple devices. Each connection can be updated and deleted.
+
```

Comment:
Please add a link to the documentation: "For details, refer to []()".

Reply:
Done.